docs/dietpi_sbc.md

```diff
@@ -471,9 +471,117 @@ The Hyper-V virtual machine is great for those occasions where SBC performance j
+- Desktop images should work, however, the minimal the image, the quicker the installation, as less packages will need to be removed.  
+  Remark: We do not support Ubuntu, or have any plans to do so.
+
+**Requirement 2:** Packages which have to be installed in advance:
+
+- Needed packages should already exist on most systems, however pure minimal images may require the following installations:
+
+    ```sh
+    apt update
+    apt install -y systemd-sysv ca-certificates sudo curl locales
+    ```
+
+### Script execution
+
+!!! important "Needed: Run script within SSH command or local term"
+    Do not run the script within any desktop environment, because the desktop is uninstalled during the script execution.  
+    That will "pull the ground from under your feet".
+
+!!! important "Needed: Root rights"
+    Ensure you have elevated privileges (e.g.: login as `root`, or use `sudo`).
+
+The following procedure is explained with the use of the `master` branch. Unless otherwise noted, the procedure is identical to the `beta` and the `dev` branches.
+
+Execute the following shell command to get the script (`master` branch):
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/MichaIng/DietPi/master/PREP_SYSTEM_FOR_DIETPI.sh -o PREP_SYSTEM_FOR_DIETPI.sh
+```
+
+In case of using the `beta` branch execute:
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/MichaIng/DietPi/beta/PREP_SYSTEM_FOR_DIETPI.sh -o PREP_SYSTEM_FOR_DIETPI.sh
+```
+
+In case of using the `dev` branch execute:
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/MichaIng/DietPi/dev/PREP_SYSTEM_FOR_DIETPI.sh -o PREP_SYSTEM_FOR_DIETPI.sh
+```
+
+As the next step, execute
+
+```sh
+chmod +x PREP_SYSTEM_FOR_DIETPI.sh
+./PREP_SYSTEM_FOR_DIETPI.sh
+```
+
+In the following dialog you have to select the DietPi installer branch. Choose the same branch as the DietPi `PREP_SYSTEM_FOR_DIETPI.sh` script:
+
+![DietPi-PREP branch selection](assets/images/dietpi-prep-selectbranch.png){: style="width:550px"}
+
+In the following dialogs enter your name and afterwards the actual image base and the device (SBC or PC) the system is running on:
+
+![DietPi-PREP pre-image entry](assets/images/dietpi-prep-preimage.png){: style="width:550px"}
+
+![DietPi-PREP device selection](assets/images/dietpi-prep-deviceselection.png){: style="width:550px"}
+
+Depending on whether you want to use the WiFi feature later on, you have to select the option to keep or purge the WiFi package. To keep the package could be the case if your hardware has an onboard WiFi or you add the WiFi e.g. via an USB WiFi adapter.
+
+![DietPi-PREP WiFi selection](assets/images/dietpi-prep-wifiselection.png){: style="width:550px"}
+
+The last selection is the target Debian version (the actual **Buster** or the upcoming **Bullseye**).  
+For the older version **Jessie** you need to use a different branch (branch `jessie` instead of `master`, `beta`, or `dev`).  
```

Comment:
Ah, we should remove that. Jessie is not officially supported anymore and we should not encourage users to use outdated (in case insecure) software.

Reply:
Ok, deleted.